src/components/App.js

```diff
@@ -21,7 +21,11 @@ const App = () => {
       },
       Experience: [],
       Education: [],
-      Awards: [],
+      Awards: {
+        Award_Name: "your award name",
+        Recieved : "Recieved year",
+        Description: "short description of your award",
+      },
```

Comment:
What if a person has got multiple awards?

Reply:
Yes sir , i thought about that later ,give some time i will update it

Reply:
@Amandev02 Will wait for it. But can you first discuss?

Reply:
I was thoughting to get the no of awards and implement a loop n times

Reply:
No no, that's not the right way. Check how the others are implemented. Have you worked with React?

Reply:
Or can u suggest something

Reply:
We just need to know what all the different properties, a single object can contain. That's all.

Reply:
i m new bie in react let me try